openmc/universe.py

```diff
@@ -519,3 +494,41 @@ def create_xml_subelement(self, xml_element):
                 # Append the Universe ID to the subelement and add to Element
                 cell_element.set("universe", str(self._id))
                 xml_element.append(cell_element)
+
+    def _determine_paths(self, path=''):
+        """Count the number of instances for each cell in the universe, and
+        record the count in the :attr:`Cell.num_instances` properties."""
+
+        univ_path = path + 'u{}'.format(self.id)
+
+        for cell in self.cells.values():
```

Comment:
Yes, I checked for a variety of situations including making sure that the summary.h5 distribcell paths for BEAVRS matched those I was generating from the Python API. Also checked hexagonal lattices.